easybuild/easyconfigs/q/Qhull/Qhull_pkgconfig.patch

```diff
@@ -0,0 +1,18 @@
+inspired by qhull-2003.1-pkgconfig.patch, available from git://pkgs.fedoraproject.org/qhull
```

Comment:
well, I would just add it to the configure script itself...

however, this easyconfig is used `CMake`, and I flat-out refuse to patch anything in CMake*txt ;)